### PR TITLE
Updated Vorstand page for new term

### PR DESCRIPTION
**Pull request by alanderex**
Updated after election 2023.
I used the LinkedIn pictures of Jessica and Paloma from LinkedIn, guess that'll be the right one to use. 😄

### CODE REVIEW

**Comment by kbr** [comment]
LGTM but will leave approving to Jessica/Paloma, because of the images.

**Review by sleepypioneer — COMMENTED** [review]
I'm happy with that image, thank you for checking! Will leave it for @discombobulateme to approve the PR once she checks the image :)

File: website/databags/board.json
Comment (sleepypioneer) [inline website/databags/board.json:51]: do we want to add these social media links? I'm not on Xing and barely on Twitter (now X) but my LinkedIn is https://www.linkedin.com/in/jessica0greene/ maybe folks also have personal websites?

Comment (alanderex) [reply website/databags/board.json:51]: We can add it but currently it does not show on the website. I think I can add links.

**Comment by alanderex** [comment]
Happy to report social links have been added 🚀

File: website/content/blog/mv-2023/contents.lr
Comment (sleepypioneer) [inline website/content/blog/mv-2023/contents.lr:107]: thanks for this @alanderex, great translation and summarisation, I might actually use this if I need a German bio in the future <3

File: website/content/blog/mv-2023/contents.lr
Comment (sleepypioneer) [inline website/content/blog/mv-2023/contents.lr:101]:
```suggestion
Jessica Greene hat als Quereinsteigerin eine unkonventionelle Karriere in der Technikbranche eingeschlagen, wobei die
```

**Comment by sleepypioneer** [comment]
> Happy to report social links have been added 🚀
> 
> * linkedin
> * twitter
> * github
> * website
> 
> please feel free to add your in`website/databags/board.json` and make a PR
> 
> @kbr please review the post as well `website/content/blog/mv-2023/contents.lr`
> 
> I added the staging website as PR template to make it easier 😄 http://pysv-web-staging.s3-website.eu-central-1.amazonaws.com

awesome, once this merges I will add my other links. Thank you @alanderex!

File: website/content/blog/mv-2023/contents.lr
Comment (kbr) [inline website/content/blog/mv-2023/contents.lr:92]: Klaus Bremer -> Dr. Klaus Bremer
(if titles, put them all in :)

File: website/content/blog/mv-2023/contents.lr
Comment (kbr) [inline website/content/blog/mv-2023/contents.lr:43]: Typo: beim -> bei

File: website/content/blog/mv-2023/contents.lr
Comment (kbr) [inline website/content/blog/mv-2023/contents.lr:39]: zu der mehr als 200 Teilnehmer aus verschiedenen Ländern anreisten. (- Verdoppelung von zog und anzog)

**Review by kbr — CHANGES_REQUESTED** [review]
Still found some typos and minor improvements.

**Comment by discombobulateme** [comment]
Thank you for having all this work :) I can add or change later and create a new PR, for now its really great! :)

File: website/content/blog/mv-2023/contents.lr
Comment (alanderex) [reply website/content/blog/mv-2023/contents.lr]: fixed, tx!

File: website/content/blog/mv-2023/contents.lr
Comment (alanderex) [reply website/content/blog/mv-2023/contents.lr:106]: Sure, feel free to use it! 🤗

**Pull request merged**
